docs/cel2/l2-operator-guide.md

```diff
@@ -146,7 +125,7 @@ wget https://storage.googleapis.com/cel2-rollup-files/alfajores/rollup.json --ou
 echo openssl rand -hex 32 > ${OP_NODE_DIR}/jwt.txt
 ```
 
-2. Run the container (or the binary if preferred). You can use the following example as a reference. If you're using snap sync mode, you need to add the flag `--syncmode=consensus-layer`.
```

Comment:
It looks like this is not on top of the latest version because this is a typo that's fixed in the deployed version

Reply:
Yeah, I messed up with a merge conflict